locations/spiders/nationwide_gb.py

```diff
@@ -22,4 +22,11 @@ def post_process_item(self, item, response, ld_data, **kwargs):
         if "phone" in item and item["phone"] is not None and item["phone"].replace(" ", "").startswith("+443"):
             item.pop("phone", None)
 
+        if (
+            "image" in item
+            and item["image"] is not None
+            and "https://www.bhf.org.uk/-/media/images/social/facebook-share-image_v5-1.png" in item["image"]
+        ):
```

Comment:
Can we simplify this to

```suggestion
        if item.get("image") == "https://www.bhf.org.uk/-/media/images/social/facebook-share-image_v5-1.png":
```

Reply:
no, as it has bunch of FB garbage appended to it that looks unstable (tracking parameter?)

Reply:
https://www.bhf.org.uk/-/media/images/social/facebook-share-image_v5-1.png?rev=63f70ab8b3b148edaef074404a4874c1

Reply:
How about `"social/facebook-share-image_v5-1.png" in item.get("image")` then?

Reply:
should work, unlikely to have higher false positive ratio